Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs

```diff
@@ -48,7 +50,35 @@ public partial class SpatialMappingManager : Singleton<SpatialMappingManager>
         /// <summary>
         /// The current source of spatial mapping data.
         /// </summary>
-        public SpatialMappingSource Source { get; private set; }
+        public SpatialMappingSource Source
+        {
+            get { return source; }
+
+            private set
+            {
+                if (source != value)
+                {
+                    UpdateRendering(false);
+
+                    var oldSource = source;
```

Comment:
This one I'd like to keep as "var".  The the exact type isn't important to the algorithm here, since the object is just copied in whole. How do others feel?